src/test/java/be/seeseemelk/mockbukkit/entity/BoggedMockTest.java

```diff
@@ -0,0 +1,71 @@
+package be.seeseemelk.mockbukkit.entity;
+
+import be.seeseemelk.mockbukkit.MockBukkit;
+import be.seeseemelk.mockbukkit.ServerMock;
+import net.kyori.adventure.sound.Sound;
+import org.bukkit.entity.EntityType;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.UUID;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class BoggedMockTest
```

Comment:
Is there a reason `@ExtendWith(MockbuntuExtension.class)` is not used here?

Reply:
I did not include it, because i'm using the setUp and tearDown method to mock and unmock.

Reply:
Please use them going forward, as to keep the Codebase as consitent as possible